services/client/src/pages/events/index.js

```diff
@@ -30,8 +32,13 @@ const mapDispatchToProps = function(dispatch) {
   };
 };
 
-const eventsUrl = `${process.env.REACT_APP_EVENTS_SERVICE_URL}/events`;
-const PaginatedEvents = withPagination(eventsUrl)(Events);
+const urlObject = {
+  url: `${process.env.REACT_APP_EVENTS_SERVICE_URL}/events`,
+  params: {
+    location: "dublin"
```

Comment:
I harcoded this value to test the whole functionality. How should I retrieve this from the store? Currently, the store is not being exported from its module. Should I export it and then use it here directly?